### PR TITLE
Code example: fig 5 tables and trees

### PR DESCRIPTION
**Pull request by ashander**
when checking out #45 I thought it'd be cool to see the trees in msprime

if there's interest I could polish this a bit to include as a very brief supplement; or
maybe just link to it in the readme here

### CODE REVIEW

File: examples/nodes.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+id	time	is_sample
+0	0.0	1
+1	0.0	1
+2	1.0	1
```
Comment (jeromekelleher) [inline examples/nodes.txt]: Shouldn't 2,3 and 4 be non-samples? This is causing the trees to print out in a funny way below.

**Comment by jeromekelleher** [comment]
This is great, thanks @ashander! I was wondering exactly the same thing myself. It's a useful thing to have, but I'm not sure it's worth making into a supplement. Maybe just mention it in the README as you say?

File: examples/nodes.txt (hunk 1 above)
Comment (ashander) [reply examples/nodes.txt]: I had tried that initially but it didn't draw the tree (2 (0, 1)): 97cc3b91d0be83f91b86078696371837e676cd12

Comment (ashander) [reply examples/nodes.txt]: Er, never mind I made a mistake here

Comment (ashander) [reply examples/nodes.txt]: Good call. Though: it is nice that if you mark them all as samples you can see the variation in tree height depicted in fig5c

**Comment by ashander** [comment]
README sounds good. 

See the latest update: I added a comparison of the output from ts.simplify to the trees from the table in fig5c

**Comment by jeromekelleher** [comment]
LGTM

**Comment by ashander** [comment]
Oops was planning to squash all those ugly `fixup! ...` commits before merging. nbd though :)

**Pull request merged**